<ui version="4.0" >
 <class>searchCiteseerx</class>
 <widget class="QWidget" name="searchCiteseerx" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Search CiteSeerX</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLineEdit" name="searchLine" />
   </item>
   <item row="1" column="0" colspan="6" >
    <widget class="QTableWidget" name="searchResults" />
   </item>
   <item row="0" column="3" >
    <widget class="QPushButton" name="addButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>&amp;Add</string>
     </property>
     <property name="icon" >
      <iconset resource="../icons.qrc" >
       <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="searchButton" >
     <property name="text" >
      <string>&amp;Search</string>
     </property>
     <property name="icon" >
      <iconset resource="../icons.qrc" >
       <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QPushButton" name="citedInButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Cited &amp;in</string>
     </property>
     <property name="icon" >
      <iconset resource="../icons.qrc" >
       <normaloff>:/icons/cited_in.png</normaloff>:/icons/cited_in.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QPushButton" name="citationsButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>&amp;Citations</string>
     </property>
     <property name="icon" >
      <iconset resource="../icons.qrc" >
       <normaloff>:/icons/citations.png</normaloff>:/icons/citations.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="lookUpDOI" >
     <property name="text" >
      <string>&amp;Lookup DOI</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections/>
</ui>
